src/components/tabs/themes/tab.base.scss

```diff
@@ -53,3 +65,30 @@
         display: flex;
     }
 }
+
+[part~='tab-body'] {
+    display: none;
+    position: sticky;
+    inset-inline-start: 0;
+    width: var(--tabs-width);
+    grid-row: 3;
+    grid-column: 1 / -1;
+    max-width: var(--_ig-tabs-width);
```

Comment:
I cannot see where this CSS variable is declared and the naming is somewhat confusing considering it's so close to the one a few lines above named --tabs-width.

Reply:
The value for --_ig-tabs-width is dynamically calculated in TypeScript and represents the width of the ig-tabs component. It is required to ensure the position: sticky behavior for the body works properly. Let me know if I need to change the name, for now, I will add a comment to explain its purpose.